axiom/nr_analytics_events.c

```diff
@@ -201,3 +201,10 @@ const char* nr_analytics_events_get_event_json(nr_analytics_events_t* events,
 
   return nr_analytics_event_json(events->events[i]);
 }
+
+bool nr_analytics_events_is_sampling(nr_analytics_events_t* events) {
```

Comment:
Check for events being valid (NULL, etc?).

Reply:
What should be the return value if `events` is NULL?

Reply:
I would imagine FALSE as it isn't sampling?

Reply:
Addressed in a52cca63b84671b84c650ecda18481d38f4defa8